plugins/lyrics-genius/back.js

```diff
@@ -7,7 +7,7 @@ const fetch = require("node-fetch");
 
 const { cleanupName } = require("../../providers/song-info");
 const { injectCSS } = require("../utils");
-let eastAsianChars = new RegExp("[\u{3040}-\u{30ff}\u{3400}-\u{4dbf}\u{4e00}-\u{9fff}\u{f900}-\u{faff}\u{ff66}-\u{ff9f}]");
+let eastAsianChars = new RegExp("[\uac00-\ud7af]|[\u1100-\u11ff]|[\u3130-\u318f]|[\ua960-\ua97f]|[\ud7b0-\ud7ff]|[\u3040-\u309F]|[\u3000-\u303F\u3400-\u4DBF\u4E00-\u9FFF]");
```

Comment:
```suggestion
let eastAsianChars = new RegExp("\p{Script=Han}|\p{Script=Katakana}|\p{Script=Hiragana}");
```

Reply:
oh that's cool; lemme take a look to see if it will work

Reply:
also do you know where one might find the scripts for Hangul? Korean characters are part of that regex as well

Reply:
Oh nevermind found it; I guess we can omit Kanji since it overlaps with Chinese most cases and chances are extremely high Japanese lyrics will have majority Hiragana

Reply:
Hmm seems to be an invalid regex string, maybe I'm doing it wrong. 

On a side note I was checking the second link you gave me and it said that Js doesn't have Unicode support through its RegExp class but does support \uFFFF for matching; does that have anything to do with this?

Reply:
> On a side note I was checking the second link you gave me and it said that Js doesn't have Unicode support through its RegExp class but does support \uFFFF for matching; does that have anything to do with this?

check the first link lol....

> Hmm seems to be an invalid regex string, maybe I'm doing it wrong.

That's because you are using `new RegExp` with a string instead of writing a regex literal
so, we need to escape the <kbd>\\</kbd>
I've updated my suggestion

Reply:
ohh that explains it; thanks! 

What's the difference between doing Script= and just the \p{script name} ?

Reply:
If you look at the mozilla docs I sent, it says that in javascript you can only do `\p{Script=<NAME>}`

it has to be exactly `Script=` and not `script=`